CMakeLists.txt

```diff
@@ -207,7 +209,7 @@ ENDIF()
 set(RTABMAP_QT_VERSION AUTO CACHE STRING "Force a specific Qt version.")
 set_property(CACHE RTABMAP_QT_VERSION PROPERTY STRINGS AUTO 4 5)
 
-FIND_PACKAGE(OpenCV REQUIRED QUIET)
+FIND_PACKAGE(OpenCV 3 REQUIRED QUIET)
```

Comment:
Remove hard-coded 3. IF you have OpenCV 2 or OpenCV4 also installed, use `-DOpenCV_DIR=/path/to/OpenCVConfig.cmake/directory` to select your opencv 3 installation.